lib/site_prism/dsl.rb

```diff
@@ -36,6 +36,14 @@ def raise_if_block(obj, name, has_block, type)
       raise SitePrism::UnsupportedBlockError
     end
 
+    # Prevent users from naming the elements starting with no_
+    def raise_if_name_unsupported(obj, name)
+      if name.to_s.start_with?("no_")
+        SitePrism.logger.error("#{obj.class}#{name} should not start with no_")
```

Comment:
This message is good but we should add an extra `#` symbol in between as it is an instance method that is causing problems.